src/components/contact-us/questions/_security_send_method.njk

```diff
@@ -1,3 +1,22 @@
+{% set countryHtml %}
+{{ govukInput({
+    id: "country",
+    name: "country",
+    value: country,
```

Comment:
Because this is a new custom field, I also think we'll need to add this in to the data that is sent to [zendesk](https://github.com/govuk-one-login/authentication-frontend/blob/main/src/components/contact-us/contact-us-controller.ts#L445-L475) and  [smart agent](https://github.com/govuk-one-login/authentication-frontend/blob/main/src/components/contact-us/contact-us-controller.ts#L391-L418), and make sure the question text goes through as well. 

It'd be good to get a test to actually test this out, I tried for a small amount of time yesterday.

It'd be good to get someone with zendesk access to make a trial request and make sure this is formatted correctly.

Reply:
I am in agreement with @BeckaL , but this should be raised under a different ticket, since nothing has been mentioned in the ticket https://govukverify.atlassian.net/browse/AUT-1356 about this extra functionality

Reply:
@VladGavrilet are you happy to raise a ticket with the new information we want to send to contact centre and bring it up in standup on Monday? I agree this should have been raised separately and it should have been progressed earlier as it's cross team. Not sure we want to approve + merge this change until timeline is discussed (by Vin?) with the relevant teams and the integration work is already in place.

Reply:
@mark-cab I've raised the following US, https://govukverify.atlassian.net/browse/AUT-1939 I'll bring it up on Monday

Reply:
Sorry the change I'm requesting here is not cross team, it's in this application and is just about making sure the frontend change is what we send to zendesk. The work on the other side is being discussed separately / handled. The work I'm talking about is just the completion of this form. While it wasn't literally mentioned on the ticket, it's entirely part of completing it on our end.

Reply:
Also a clarification on the testing point, that I just mean that when Gwyn spins the app up locally he has it set up to submit to zendesk, so he can submit tickets locally and see them go through to zendesk, so is just a useful end to end check that everything is working. I think other people in our team should be able to do this but I don't have access to the zendesk queues at the moment.

Reply:
Here's an [example commit](https://github.com/govuk-one-login/authentication-frontend/pull/1187/commits/1b711a2bf585254ef0054d87c861bafc8fbc4f9a) of what I mean from my PR. As you can see, it's just some stuff that's chucked into some html, so doesn't require changes across teams. I should say that I'm not the domain expert here, but this is my understanding of what we need to do.